### ISSUE TITLE
Still works?

### ISSUE THREAD

**Issue by eikum** [amosyuen/ha-mikucare#77]
Does this application still work? I have some problems, not sure if it's the new subscription system fault or not.

**Comment by amosyuen** [comment]
This repository is no longer maintained. I do not own a miku anymore so I cannot test the repository.

**Comment by camthekeeper1** [comment]
No, unfortunately this does not work still.  Thank you @Amosyuen for creating this initially.

**Comment by trevoreduke** [comment]
I am having a baby soon, and this kind of integration is exactly what I am looking for. 

If I paid for half your Miku, would that be enough to get development going again @amosyuen ?

**Comment by amosyuen** [comment]
Apologies, but I have too many other things going on right now so have no plans to pick up this integration again even if I had the hardware.
